compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -528,10 +539,14 @@ void buildAMDAIETransformPassPipeline(OpPassManager &variantPassManager) {
         [&]() { return createAMDAIELowerExecutableTargetPass(options); });
   }
   modulePassManager.addPass(createLowerUKernelOpsToCallsPass());
-  if (clUsePipeline == AIEPassPipeline::PadPackPipeline) {
-    addMLIRAIRLoweringPasses(modulePassManager, false);
-  } else if (clUsePipeline == AIEPassPipeline::PackPeelPipeline) {
-    addMLIRAIRLoweringPasses(modulePassManager, true);
+  if (clUseBackend == AIEBackend::ObjectFifo) {
+    addAMDAIEObjectFifoLoweringPasses(modulePassManager);
+  } else if (clUseBackend == AIEBackend::AIR) {
+    if (clUsePipeline == AIEPassPipeline::PadPackPipeline) {
+      addMLIRAIRLoweringPasses(modulePassManager, false);
+    } else if (clUsePipeline == AIEPassPipeline::PackPeelPipeline) {
+      addMLIRAIRLoweringPasses(modulePassManager, true);
+    }
   }
```

Comment:
2 design change suggestions
```
if (clUseBackend == AIEBackend::ObjectFifo) {
    addAMDAIEObjectFifoLoweringPasses(modulePassManager);
 } else if (clUseBackend == AIEBackend::AIR) {
     bool isPackPeel = clUsePipeline == AIEPassPipeline::PackPeelPipeline;
     addMLIRAIRLoweringPasses(modulePassManager, isPackPeel);
 } else{ 
    assert(false, "Only 2 backends supported here"); 
 }
 ```

Reply:
This wasn't needed anymore after the addition of the `ConvDecompose` pipeline: https://github.com/nod-ai/iree-amd-aie/pull/487/files#diff-42f0d0bb098689f25ee68e8f05ec6c2eaa89ce41a6394d7d99c1d1c912943b38R548